package.json

```diff
@@ -18,72 +18,86 @@
-    "@wordpress/data": "4.11.0",
-    "@wordpress/dependency-extraction-webpack-plugin": "2.1.0",
-    "@wordpress/dom": "2.8.0",
-    "@wordpress/editor": "9.9.0",
-    "@wordpress/element": "2.10.0",
+    "@testing-library/jest-dom": "5.5.0",
+    "@testing-library/react": "10.0.3",
+    "@wordpress/api-fetch": "3.14.0",
+    "@wordpress/babel-preset-default": "4.12.1",
+    "@wordpress/block-editor": "3.10.0",
+    "@wordpress/components": "9.5.0",
+    "@wordpress/compose": "3.14.0",
+    "@wordpress/core-data": "2.15.0",
+    "@wordpress/data": "4.17.0",
+    "@wordpress/dependency-extraction-webpack-plugin": "2.6.0",
+    "@wordpress/dom": "2.9.0",
+    "@wordpress/e2e-test-utils": "4.6.0",
+    "@wordpress/e2e-tests": "1.15.0",
+    "@wordpress/editor": "9.15.0",
+    "@wordpress/element": "2.13.1",
+    "@wordpress/env": "1.3.0",
     "@wordpress/eslint-plugin": "3.3.0",
     "@wordpress/hooks": "2.8.0",
-    "@wordpress/html-entities": "2.5.0",
-    "@wordpress/i18n": "3.7.0",
-    "@wordpress/keycodes": "2.7.0",
-    "@wordpress/scripts": "6.1.1",
-    "@wordpress/server-side-render": "1.5.0",
-    "@wordpress/url": "2.13.0",
-    "autoprefixer": "9.7.3",
+    "@wordpress/html-entities": "2.7.0",
+    "@wordpress/i18n": "3.12.0",
+    "@wordpress/jest-console": "3.6.0",
+    "@wordpress/keycodes": "2.12.0",
+    "@wordpress/scripts": "9.0.0",
+    "@wordpress/server-side-render": "1.11.0",
+    "@wordpress/url": "2.14.0",
+    "autoprefixer": "9.7.6",
     "babel-core": "6.26.3",
     "babel-eslint": "10.1.0",
-    "babel-loader": "8.0.6",
-    "cross-env": "6.0.3",
+    "babel-loader": "8.1.0",
+    "classnames": "2.2.6",
+    "cross-env": "7.0.2",
     "css-loader": "3.4.0",
-    "del": "^5.1.0",
+    "del": "5.1.0",
+    "enzyme": "3.11.0",
     "eslint": "6.7.2",
     "eslint-plugin-eslint-comments": "3.1.2",
     "eslint-plugin-import": "2.20.2",
     "eslint-plugin-jsx-a11y": "6.2.3",
-    "gulp": "^4.0.2",
-    "gulp-run": "^1.7.1",
-    "gulp-string-replace": "^1.1.2",
-    "husky": "^4.0.10",
-    "jest-silent-reporter": "0.1.2",
-    "merge-stream": "^2.0.0",
-    "mini-css-extract-plugin": "0.8.2",
-    "node-sass": "^4.13.1",
+    "gulp": "4.0.2",
+    "gulp-run": "1.7.1",
+    "gulp-string-replace": "1.1.2",
+    "husky": "4.2.5",
+    "jest-silent-reporter": "0.2.1",
+    "lodash": "4.17.15",
+    "merge-stream": "2.0.0",
+    "mini-css-extract-plugin": "0.9.0",
+    "node-sass": "4.13.1",
     "npm-run-all": "4.1.5",
-    "postcss-loader": "^3.0.0",
-    "raw-loader": "^4.0.0",
+    "postcss-loader": "3.0.0",
+    "raw-loader": "4.0.1",
     "re-resizable": "6.3.2",
-    "react": "16.12.0",
-    "sass-loader": "^8.0.2",
-    "style-loader": "^1.0.2",
-    "uglifyjs-webpack-plugin": "^2.2.0",
-    "webpack": "^4.41.3",
-    "webpack-cli": "3.3.10"
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
+    "sass-loader": "8.0.2",
+    "style-loader": "1.0.2",
+    "uglifyjs-webpack-plugin": "2.2.0",
```

Comment:
The leading `^` is removed, as it should help our builds be similar, even in our different locals.